### PR TITLE
Code system examples

### PR DESCRIPTION
**Pull request by Shaumik-Ashraf**
Create json examples for all code systems as specified in IG and placed in `examples/byCodeSystem`. This addresses issue #153 and ticket DVCI-99. I also added ATC in `profiles_vaccination.fsh`, and moved Scenario3Bundle and some examples to `tests_errors.csv` to pass maven test (which fails because external terminology servers are inactive). See `examples/README.md` for details.

### CODE REVIEW

File: examples/byCodeSystem/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+| :-------: | :------------------------------------ | :----------------: | :--------------------: |
+| GTIN      | <https://www.gs1.org/gtin>            | VaccineProductGTIN | NO                     |
+| CVX       | <http://hl7.org/fhir/sid/cvx>         | VacccineProductCVX | NO (for Covid)         |
+| AIR       | <https://www.humanservices.gov.au/organisations/health-professionals/enablers/air-vaccine-code-formats> | Austrailian Immunisation Register Vaccine | NO |
+| SNOMED CT | <http://snomed.info/sct>              | VaccineTypeSNOMED  | YES                    |
+| ICD-11    | <http://id.who.ind/icd11/mms>         | VaccineTargetICD11 | YES                    |
+| ATC/DDD   | <https://www.whocc.no/atc_ddd_index/> | VaccineTargetATC   | YES                    |
+
+
+## Approved Vaccines according to [Regulatory Affiars Professionals Society](https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker)
```
Comment (mickohanlon23) [inline examples/byCodeSystem/README.md:19]:
```suggestion
## Approved Vaccines according to [Regulatory Affairs Professionals Society](https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker)
```

**Review by mickohanlon23 — COMMENTED** [review]
All of the example resources pass the Inferno FHIR Validator accept for the value set issue raised in `examples/README.md` as well as the invalid ATC slice, which was just added in this PR.

**Comment by p9g** [comment]
At this point is there any benefit to using FSH to generate the examples? (Used gofsh to create the FSH from the json)

File: examples/byCodeSystem/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+| Vaccine           | GTIN            | CVX      | AIR      | SNOMED CT | ICD-11    | ATC/DDD  |
+| :---------------: | :-------------: | :------: | :------: | :-------: | :-------: | :------: |
+| Pfizer            |                 | 208      | COMIRN   | 840534001 |  XM0GQ8   | J07BX03  |
+| Moderna           | 30380777700688  | 207      |          |   "   "   |  XM0GQ8   |  "   "   |
+| AstraZeneca       |                 | 210      | COVAST   |   "   "   |  XM9QW8   |  "   "   |
+| Sputnik V         |                 | 213      |          |   "   "   |  XM9QW8   |  "   "   |
+| Sputnik Light     |                 | 213      |          |   "   "   |  XM9QW8   |  "   "   |
+| Janssen           | 00359676580157  | 212      |          |   "   "   |  XM9QW8   |  "   "   |
+| Sinovac           |                 | 213      |          |   "   "   |  XMLNL1   |  "   "   |
```
Comment (p9g) [inline examples/byCodeSystem/README.md:55]: change XMLNL1 to XM1NL1
see https://icd.who.int/dev11/l-m/en#/http%3a%2f%2fid.who.int%2ficd%2fentity%2f1104808441

Comment (Shaumik-Ashraf) [reply examples/byCodeSystem/README.md]: I just fixed this in next commit, thank you for finding it

**Comment by Shaumik-Ashraf** [comment]
> At this point is there any benefit to using FSH to generate the examples? (Used gofsh to create the FSH from the json)

Cannot use FSH for reason here: https://github.com/dvci/vaccine-credential-ig/blob/main/examples/README.md

I made the examples in json from scratch

File: examples/README.md [hunk 3]
```diff
@@ -2,4 +2,35 @@
...
+| `atc_gtin_immunization`    | Patient received one dose of Janssen vaccine, specified by ATC and GTIN systems |
+| `cvx_immunization`         | Patient received two doses of an unspecified vaccine, presumably (but not necessarily) from China, Russia, or another foriegn country |
+| `gtin_immunization`        | Patient received two doses of Moderna vaccine    |
+| `icd11_immunization`       | Patient received two doses of Pfizer vaccine, manufacturer specified by MVX     |
+| `icd11_gtin_immunization`  | Patient received two doses of Moderna vaccine, specified by ICD-11 and GTIN     |
+| `snomed_immunization`      | Patient received two doses of Pfizer vaccine, manufacturer specified by GLN     |
+| `snomed_gtin_immunization` | Patient recieved two doses of Moderna vaccine, manufacturer specified by SNOMED and GTIN |
+
+
+## Validation notes:
```
Comment (nkarip) [inline examples/README.md:29]: If there are expected errors from the validator, they should be documented in the IG so implementers know to expect their tests to fail. We've been using the conformance.md page Validation section.

File: examples/README.md [hunk 4]
```diff
@@ -2,4 +2,35 @@
 
 Examples are typically generated within a FHIR Implementation Guide. However, the [FHIR Publisher](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation) responsible for building Implementation Guides places additional requirements on examples that are incompatible with this IG's [data minimization criteria](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/conformance.html#data-minimization), such as requiring `id` elements and not recognizing absolute URIs like `resource:0` for within-Bundle resolution.
 
-Until these issues with the IG Publisher are resolved, we are storing our example resources in this folder outside the IG build process, and linking from the IG to these files on GitHub.
\ No newline at end of file
+Until these issues with the IG Publisher are resolved, we are storing our example resources in this folder outside the IG build process, and linking from the IG to these files on GitHub.
```
Comment (nkarip) [inline examples/README.md:5]: Links for the new examples should be added to the examples page for the publisher to pick up.
The examples are listed at /input/data/examples.json.

**Review by nkarip — CHANGES_REQUESTED** [review]

File: examples/byCodeSystem/README.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+Also added Novavax vaccine, although it is not yet approved as of Jun 9th, 2021.
+
+## Vaccine Codes
+
+Blank means values are missing, and " " indicates same value for each row.
+
+| Vaccine           | GTIN            | CVX      | AIR      | SNOMED CT | ICD-11    | ATC/DDD  |
+| :---------------: | :-------------: | :------: | :------: | :-------: | :-------: | :------: |
+| Pfizer            |                 | 208      | COMIRN   | 840534001 |  XM0GQ8   | J07BX03  |
```
[the remark this answers is not in the document]
Comment (nkarip) [reply examples/byCodeSystem/README.md]: Fixed in latest push.

File: examples/README.md (hunk 4 above)
Comment (nkarip) [reply examples/README.md:5]: This looks good - a couple of notes.

- [ ] there is a typo in StructureDefinition-vaccination-credential-immunization-resource-examples.html 'foriegn'
- [ ] the Immunization profiles examples list should omit 'dose 1' or 'dose 2' descriptors. The immunization profiles do not include any dose number indicators, that has to be computed from the full bundle. For examples of individual immunization resources, we should leave out dose number from the description. You just need once example for each type.

**Review by nkarip — APPROVED** [review]

**Comment by p9g** [comment]
Not really related to code system examples, but would be nice if https://github.com/HL7/fhir-shc-vaccination-ig/blob/dev/input/pagecontent/contact.md contained a mailto link for sending to vaccination@lists.hl7.org

**Pull request merged**